src/main/java/org/mitre/synthea/modules/CardiovascularDiseaseModule.java

```diff
@@ -756,12 +756,26 @@ private static void atrialFibrillationTreatment(Person person, long time) {
     }
   }
 
-  private static void prescribeMedication(String med, Person person, long time, boolean chronic) {
-    Medication entry = person.record.medicationStart(time, med, chronic);
+  /**
+   * Add a prescription.
+   * @param med The medication key, corresponding to a value in the LOOKUP table.
+   * @param person The patient.
+   * @param time The time the medication was given or prescribed.
+   * @param rx If true, prescribe as a chronic medication.
+   *     If false, administer the drug immediately.
+   */
+  private static void prescribeMedication(String med, Person person, long time, boolean rx) {
```

Comment:
I renamed the method to be `giveMedication(...)` to be more ambiguous, with the `rx` parameter determining whether or not to prescribe or administer.

I don't think it makes sense to make two different methods because all the other code is the same. The `encounter.provider.incrementPrescriptions(year)` is metadata for tracking how many medications are prescribed by organization, but it isn't really being used effectively anywhere. A medication administration is still prescribed by the doctor, and immediately administered, even though the patient doesn't walk away and get it filled at a pharmacy. We could separately out these metadata if we really want to, but I'm not sure how much value it would have.